From 6a63b589bb3a2fe1820c89769af1c1b8bd9367bd Mon Sep 17 00:00:00 2001
From: Camelia Groza <camelia.groza@nxp.com>
Date: Thu, 7 Jun 2018 14:29:49 +0300
Subject: [PATCH 542/767] staging: fsl-dpaa2/eth: ceetm: make classify function
 match its prototype

Signed-off-by: Camelia Groza <camelia.groza@nxp.com>
[Xulin: Original patch taken from NXP LSDK-18.09.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/staging/fsl-dpaa2/ethernet/dpaa2-eth-ceetm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/staging/fsl-dpaa2/ethernet/dpaa2-eth-ceetm.c b/drivers/staging/fsl-dpaa2/ethernet/dpaa2-eth-ceetm.c
index 7b64b71f167f..5c1e58b99d6f 100644
--- a/drivers/staging/fsl-dpaa2/ethernet/dpaa2-eth-ceetm.c
+++ b/drivers/staging/fsl-dpaa2/ethernet/dpaa2-eth-ceetm.c
@@ -1183,7 +1183,7 @@ struct Qdisc_ops dpaa2_ceetm_qdisc_ops __read_mostly = {
 
 /* Run the filters and classifiers attached to the qdisc on the provided skb */
 int dpaa2_ceetm_classify(struct sk_buff *skb, struct Qdisc *sch,
-			 int *qdid, int *qpri)
+			 int *qdid, u8 *qpri)
 {
 	struct dpaa2_ceetm_qdisc *priv = qdisc_priv(sch);
 	struct dpaa2_ceetm_class *cl = NULL;
-- 
2.17.0

